browser_sniffer.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 1.9.3"
 
-  spec.add_development_dependency "bundler", "~> 1.3"
+  spec.add_development_dependency "bundler"
```

Comment:
@ragalie Do you know if this would be an issue? I couldn't get CI to pass without this change (ci has bundler 2.0.1, but the project expects 1.3)